File: BackendSide/Question.cs
using System;
using System.Collections.Generic;

namespace BackendSide
{
    public class Question
    {
        public string questionText { get; set; }

        public List<option> options { get; set; }
    }

    public class option
    {
        public string text;
        public string score;
    }
}

File: ClientSide/src/app/models/question.ts
export interface option {
  text: string;
  score: number;
}

export interface question {
  id: number;
  questionText: string;
  options: option[];
  type: string;
  comment: string;
}

export interface answer {
  userId: string;
  questionId: number;
  chosenOptions: option[];
  questionType: string;
  comment: string;
}

export const enum questionType {
  single,
  multiple,
}
